### PR TITLE
xion testnet registration

### PR DESCRIPTION
**Pull request by ash-burnt**

### CODE REVIEW

File: cosmos/xion-testnet.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  "chainName": "Xion Testnet",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/xion-testnet/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "XION",
+    "coinMinimalDenom": "uxion",
+    "coinDecimals": 6,
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/xion-testnet/chain.png"
+  },
+  "walletUrl": "https://wallet.keplr.app/chains/xion-testnet",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/xion-testnet",
```
Comment (editaahn) [inline cosmos/xion-testnet.json:14]: The page isn't supported by Keplr Dashboard dynamically.
You can remove these two properties from the change.

File: src/constants.ts [hunk 2]
```diff
@@ -56,4 +56,5 @@ export const nativeTestnetChainIdentifiers: string[] = [
   "test-core",
   "govgen",
   "seda-1-testnet",
+  "xion-testnet",
```
Comment (editaahn) [inline src/constants.ts:59]: Could you remove this line? The registry doesn't require it.

Comment (ash-burnt) [reply src/constants.ts]: Done

File: cosmos/xion-testnet.json (hunk 1 above)
Comment (ash-burnt) [reply cosmos/xion-testnet.json]: Done

File: cosmos/xion-testnet.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    }
+  ],
+  "feeCurrencies": [
+    {
+      "coinDenom": "XION",
+      "coinMinimalDenom": "uxion",
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/xion-testnet/chain.png",
+      "gasPriceStep": {
+        "low": 0.0,
```
Comment (editaahn) [inline cosmos/xion-testnet.json:39]: Let me do one more check. Is it okay for your chain and nodes to address zero gas?

**Comment by editaahn** [comment]
Oh, you should offer information for reaching you, by adding `nodeProvider`.
```
  "nodeProvider": {
    "name": "",
    "email": "",
    "website": ""
  }
```

File: cosmos/xion-testnet.json (hunk 3 above)
Comment (editaahn) [reply cosmos/xion-testnet.json:39]: @ash-burnt

File: cosmos/xion-testnet.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      }
+    }
+  ],
+  "features": [
+    "cosmwasm"
+  ],
+  "nodeProvider": {
+    "name": "Dylan Schultzie",
+    "email": "security@lavenderfive.com",
+    "website": "lavenderfive.com"
```
Comment (editaahn) [inline cosmos/xion-testnet.json:51]: Please prepend a scheme(https://) to the url

File: cosmos/xion-testnet.json (hunk 3 above)
Comment (ash-burnt) [reply cosmos/xion-testnet.json]: Updated to 0.005, will change to zero if/when validators verify

**Pull request merged**